### DISCUSSION TITLE
Dataset and training

### DISCUSSION THREAD

**Discussion by kwilliams81** [AIRLegend/aitrack#83] [Q&A] [answered]
What are you using for you training set?

**Comment by AIRLegend** [comment, marked as answer]
Check out this project: https://github.com/emilianavt/OpenSeeFace

**Reply by kwilliams81** [reply]
Interesting, so for what I assume your project is using it was trained with LS3D-W and the WFLW dataset?

**Reply by AIRLegend** [reply]
Yes! The models used her have been trainded by emilianavt and custom C++ code for inference has been developed. Prior to that, I trained other models myself (and also tried other projects) but I found the ones there to perform better.

**Comment by Toshiwoz** [comment]
And is it possible for someone that is not expert to train based on my own setup?
I have a strong light source coming from one side, and it makes me look like 2 faces, and AITrack doesn't like it.
I can program in C# but I have almost zero experience in C++.
